content/docs/quickstart/index.md

```diff
@@ -1,41 +1,246 @@
-   Open your web browser and navigate to the Red Hat Developer Hub application. Retrieve the URL using the following OpenShift CLI command.
+There are two basic types of notifications:
 
-   ```bash
-   oc get route backstage-backstage -n rhdh-operator -o jsonpath='{.spec.host}'
-   ```
+- **Broadcast**: Messages sent to all users of Backstage.
+- **Entity**: Messages delivered to specific listed entities, such as Users or Groups.
 
-   Make sure the route is accessible to you locally.
+Example of use-cases:
 
-4. **Login to backstage**
-   Login to backstage with the Guest account.
+- System-wide announcements or alerts
+- Notifications for component owners: e.g., build failures, successful deployments, new vulnerabilities
+- Notifications for individuals: e.g., updates you have subscribed to, new required training courses
+- Notifications pertaining to a particular entity in the catalog: A notification might apply to an entity and the owning team.
 
-5. **Navigate to Orchestrator**:
-   Navigate to the Orchestrator page by clicking on the Orchestrator icon in the left navigation menu.
-   ![orchestratorIcon](./orchestratorIcon.png)
+## Installation
 
-6. **Execute Greeting Workflow**:
-   Click on the 'Execute' button in the ACTIONS column of the Greeting workflow.
-   ![workflowsPage](./workflowsPage.png)
-   The 'Run workflow' page will open. Click 'Next step' and then 'Run'
-   ![executePageNext](./executePageNext.png)
-   ![executePageRun](./executePageRun.png)
-7. **Monitor Workflow Status**:
-   Wait for the status of the Greeting workflow execution to become _Completed_. This may take a moment.
-   ![workflowCompleted](./workflowCompleted.png)
+Both the Notifications and Signals plugins are installed along the Orchestrator by the Helm chart.
+
+## Configuration
+
+### Notifications Backend
+
+The Notifications backend plugin provides an API to create notifications, list notifications per logged-in user, and search based on parameters.
+
+The plugin uses a relational [database](https://backstage.io/docs/getting-started/config/database) for persistence, no specifics are introduced in this context.
+
+No additional configuration in the app-config is needed, except for optional additional modules for `processors`.
+
+### Notifications Frontend
+
+The recipients of notifications have to be entities in the catalog, e.g. of the User or Group kind.
+
+Otherwise no specific configuration is needed for the front-end notifications plugin.
+
+All parametrization is done through component properties, such as the `NotificationsSidebarItem`, which can be used as an active left-side menu item in the front-end (see values in the Helm chart).
+
+![Notifications Page](notificationsPage.png)
+
+## Use
+
+New notifications can be sent either by a backend plugin or an external service through the REST API.
+
+### Backend
+
+Regardless of technical feasibility, a backend plugin should avoid directly accessing the notifications REST API.
```

Comment:
I think those technical details aren't relevant for this context, in which two aspects are relevant:
1. How to configure the plugins 
2. How to use the plugin

We can add a reference to the plugin's README on backstage, but no need for the details here, since we'd like to install and use the plugin on OCP.